### ISSUE TITLE
names conflict error

### ISSUE THREAD

**Issue by l0rda** [se-panfilov/vue-notifications#20]
Hello,

when i'm using 2 or more instances of one vue component with notifications block, i got error in console:

> VueNotifications: names conflict - showMyCustomerError

**Comment by se-panfilov** [comment]
Hello, @l0rda 
Thanks for your feedback.

To be honest, I'm not sure if I'm able to fix this issue in current version of vue-notifications.
Because by design vue-notifications shall be used on single-instanced components, such as "pages". 

But I'll take a look if I can do something with it.

P.S. Right now I'm working on [next iteration](https://github.com/se-panfilov/vue-notifications/tree/feature/next) of vue-notifictions that won't have such problems. Would be smaller and much more straight forward. 

P.P.S. Basically that error happens when component's instance already have method with such name  ("showMyCustomerError" in your case). It's looks like vue-notifications tried to add same method for the component at least twice.

**Comment by dkcwd** [comment]
Hey @se-panfilov and @l0rda, just thought I'd share that I ran into the same thing and decided to use https://www.npmjs.com/package/vue-geb to centralise notification handling.

As an example of a quick implementation:

In your "main" page
```
  created: function () {
    // listen only to events emitted with the id 'vue-notification'
    this._notificationSub = this.$geb.getFilteredBus({id: 'vue-notification'}).subscribe(this.processNotification)
  },
  destroyed: function () {
    // avoid memory leaks by removing the subscription when destroyed
    if (this._notificationSub) {
      this._notificationSub.unsubscribe()
    }
  },
  methods: {
    processNotification: function (notificationData) {
      if (typeof notificationData.payload.type === 'string') {
        this[notificationData.payload.type]()
      }
    }
  },
  // register all the required notifications
  notifications: {
    showMyCustomError: {
      title: 'Failed',
      message: 'Failed to do the thing',
      type: 'error'
    }
  },
```

In a component within a method which should trigger a notification
```
  this.$geb.emit({id: 'vue-notification', payload: {type: 'showMyCustomError'}})
```

**Comment by azhard4int** [comment]
@se-panfilov any ETA for the next iteration release?

**Comment by se-panfilov** [comment]
@azhard4int  and all other guys.

I'll try to release new version this weekend.
Sorry for delay, but I'm on the middle of job change right now.

Anyway, just to notice you - next release will be without backward compatability

**Comment by daniplaninc** [comment]
Hi, great library, been using it for a while now (+mini-toastr). No problem whatsoever, except the names conflict (I sometimes reuse the same vue components many times on the same page). Any ETA on the next version that would resolve this issue?

**Comment by se-panfilov** [comment]
@daniplaninc Can you provide an example?

Sorry I'm super busy, I've try to fix it today, but it's works for me.
Basically this kind of errors should happens only if it clashes with method names

**Comment by daniplaninc** [comment]
Sorry for a delayed answer. This is not a breaking issue for me, as far as I know, the only thing that happens is the error message that appears in the console ("VueNotifications: names conflict - <notificationName>). My application continues to work as expected.

As far as an example goes, maybe a description is enough. I have several global components defined (Vue.component(...)) and I use them repeatedly throughout my HTML, sometimes in a v-for. On page load, the error messages about duplication appear.

So basically whenever you will release a new version I will switch to get rid of the errors, but I am in no hurry.

**Comment by wa05** [comment]
Hi ! Any idea on how to solve this problem, Ive tried changing events, changing structure of app, but nothing, 

I have working the notifications in an area with children routes, when I access the parent routes and go back the notifications starts to work badly, repeating more and more...

**Comment by somuraijoe** [comment]
I receive same error, @se-panfilov do you have any news ?

**Comment by CrixuAMG** [comment]
Any news?
The console is littered with errors: [screenshot](http://imgur.com/a/5zIYs)

**Comment by se-panfilov** [comment]
Fixed in release [0.8.1](https://github.com/se-panfilov/vue-notifications/releases/tag/v0.8.1)

**Comment by CrixuAMG** [comment]
That fixed it, thanks @se-panfilov!
